scripts/setup_devmode.sh

```diff
@@ -44,9 +44,10 @@ CFG_TMPDIR = u'/tmp'
 COLLECT_STORAGE = u'flask_collect.storage.link'
 EOF
 
+echo "Building Assets, bower install"
 # fetch and build assets
 inveniomanage bower -i bower-base.json > bower.json
-bower install --silent
+bower install --quiet
```

Comment:
I think most of our logs are spilled by (mostly useless) test outputs which btw also makes it hard to read the test results.